pkg/netceptor/netceptor.go

```diff
@@ -178,12 +178,19 @@ type routingUpdate struct {
 	SuspectedDuplicate uint64
 }
 
+const (
+	typePacketConn = 0
+	typeConnNonTLS = 1
+	typeConnTLS    = 2
+)
```

Comment:
These consts should be public, since someone getting a ServiceAdvertisement can see a ConnType byte, and needs to know what it means.  Also, I think they should be named ConnTypeXXX rather than TypeXXX.

Reply:
made public, and renamed to `ConnTypeDatagram`, `ConnTypeStream`, and `ConnTypeStreamTLS`